Contents/Libraries/Shared/subliminal_patch/utils.py

```diff
@@ -55,4 +55,4 @@ def fix_inconsistent_naming(title, inconsistent_titles_dict=None):
         title = pattern.sub(lambda x: inconsistent_titles_dict[x.group()], title)
 
     # return fixed and sanitized title
-    return sanitize(title)
+    return title
```

Comment:
`return title if no_sanitize else sanitize(title)`

Reply:
and modify the function like this:
`def fix_inconsistent_naming(title, inconsistent_titles_dict=None, no_sanitize=False):`